README.md

```diff
@@ -3276,6 +3278,31 @@ config = Config.new
 
 ---
 
+### Plugins: vault
+
+- `Qonfig.plugin(:vault)`
+- adds support for `vault kv store`, [more info](https://www.vaultproject.io/docs/secrets/kv/kv-v2)
+- depends on `vault` gem ([link](https://github.com/hashicorp/vault-ruby)) (tested on `>= 0.1`);
+- provides `.load_from_vault` (works in `.load_from_yaml` manner ([doc](#load-from-yaml-file)));
+- provides `.expose_vault` (works in `.expose_yaml` manner ([doc](#expose-yaml)));
+
+```ruby
+# 1) require external dependency
+require 'vault'
+
+# 2) Setup vault client
+
+Vault.address = 'http://localhost:8200'
+Vault.token = 'super-duper-token-here'
+
+# 3) enable plugin
+Qonfig.plugin(:vault)
+
+# 3) use vault :)
+```
+
+---
+
 ## Roadmap
 
```

Comment:
it would be nice to add the `instance-based behavior for "vault" plugin` improvement part in roadmap :+1: